contracts/AssetToken.sol

```diff
@@ -72,7 +72,7 @@ contract AssetToken is Initializable, ERC223Interface, IERC20 {
     }
 
     modifier noOwnerAsCounterparty(address counterparty) {
-        if (counterparty == _owner) {
+        if (counterparty == _owner || counterparty == _fundingDelegate) {
```

Comment:
and the emergency stop delegate should also not be able to be party in transactions.

Reply:
also we should have some tests checking the delegates can not perform transactions